### PR TITLE
Added sample actor ref factory for reference

### PR DESCRIPTION
**Pull request by girishchbharti**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -43,7 +43,15 @@ object Dependencies {
...
   def kafka = Def.setting {
-    "org.apache.kafka" % "kafka_2.12" % "0.10.2.1" :: Nil
+    "org.apache.kafka" %% "kafka" % "0.10.2.1" :: Nil
+  }
+
+  def akka = Def.setting {
+    "com.typesafe.akka" %% "akka-actor" % "2.4.16" :: Nil
+  }
+
+  def akkaTestKit = Def.setting {
```
Comment (dagaharshit) [inline project/Dependencies.scala]: @girishchbharti akka version and its test kit version are different !

File: twillio/src/test/scala/com/knoldus/twillio/actors/TestActorSpec.scala [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+class LocalActorRefFactoryTest extends TestKit(ActorSystem("LocalActorRefFactoryTest")) with ImplicitSender
+  with AsyncWordSpecLike with MustMatchers with MockitoSugar with BeforeAndAfterAll  {
+
+  val mockedTestService = mock[TestService]
+
+  object TwillioActorFactoryTest extends TwillioActorFactory(mockedTestService)
+
+  "TwillioActorFactory" should {
+
+    "be able to get actor 1" in {
```
Comment (dagaharshit) [inline twillio/src/test/scala/com/knoldus/twillio/actors/TestActorSpec.scala]: @girishchbharti we can write better test cases using should or in. 
For reference, you can go through test cases of client module.

Comment (girishchbharti) [inline twillio/src/test/scala/com/knoldus/twillio/actors/TestActorSpec.scala]: @dagaharshit I am not sure if the same approach for 'test cases of client module' fits in here. The scenario can be different in different cases. I tried this rewrite the test cases with samme approach, but it doesn't seem logical to me. I am sending another PR with the approach you are suggesting. Just merge one you find better and close other one.

Comment (girishchbharti) [inline twillio/src/test/scala/com/knoldus/twillio/actors/TestActorSpec.scala]: Actually, I made changes into same branch with version updates.

File: project/Dependencies.scala (hunk 1 above)
Comment (girishchbharti) [inline project/Dependencies.scala]: Fixed

**Pull request merged**